### DISCUSSION TITLE
Strange line below bullet points

### DISCUSSION THREAD

**Discussion by jenst44** [nvim-neorg/neorg#1288] [Q&A] [answered]
<img width="581" alt="Screenshot 2024-01-25 at 4 57 19 PM" src="https://github.com/nvim-neorg/neorg/assets/12418840/17e7c1fc-2ac8-4c30-acb6-8f881cbdc1cd">

I'm currently having an issue where I'm seeing a strange line injected after bullet points. I assume this is due to another plugin that I have installed but I haven't been able to figure out what is causing it yet. Any ideas on what it might be from? Thanks so much!

**Comment by pysan3** [comment, marked as answer]
https://github.com/lukas-reineke/headlines.nvim

**Reply by jenst44** [reply]
Thank you so much that's exactly what I was looking for!
